docs/src/_reference/command-line-interface.md

```diff
@@ -116,6 +116,53 @@ meltano add extractor tap-spotify --no-install
 
 The `add` command does not run relative to a [Meltano Environment](https://docs.meltano.com/concepts/environments). The `--environment` flag and [`default_environment` setting](https://docs.meltano.com/concepts/environments#default-environments) in your `meltano.yml` file will be ignored if set.
 
+## `compile`
+
+### How to use
+
+Generally, the `compile` command need not be executed manually, as it will be run automatically by Meltano as-needed.
+
+If you wish to manually compile a manifest file for each environment (including the no-environment manifest file), the compile command can be used like so:
+
+```bash
+meltano compile
+```
```

Comment:
The CLI `--help` would indicate `.meltano/manifests` but just from this page it's not clear to which subdirectory it'll save those files.